src/components/WavesLayer/WavesLayer.css

```diff
@@ -12,3 +12,76 @@ svg {
 }
 
 /* svg { position:fixed; top:0; bottom: 206px;; left:0; right:0 } */
+
+
+/* .waves-bg {
+    width: 100%;
+    height: 960px;
+    position: absolute;
+    z-index: -4;
+    bottom: 0;
+    background: url('../../assets/svgs/wave-shayan.svg') no-repeat;
+    background-size: cover;
+} */
+/* svg {
+  height: 400px;
+} */
+
+/* svg { position:fixed; top:0; bottom: 206px;; left:0; right:0 } */
+
+
+/* 
+html, body { height: 100%; }
+/* body {
+  background:white;
+  overflow: hidden;
+} */
+
+/* .ocean { 
+  height: 7%;
+  width:100%;
+  position:absolute;
+  bottom:0;
+  left:0;
+  background: #346b84;
+}
+
+.wave {
+  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/85486/wave.svg) repeat-x; 
```

Comment:
How do you feel about implementing an svg morph for this kind of animation? Here's what I'm thinking
https://youtu.be/lPJVi797Uy0?t=538